### PR TITLE
[13.0][MIG] web_action_conditionable

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Supersedes https://github.com/OCA/web/pull/1395

### CODE REVIEW

File: web_action_conditionable/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This module was written to extend the functionality of actions in
+tree view (One2Many fields).
```
Comment (simahawk) [inline web_action_conditionable/readme/DESCRIPTION.rst]:
```suggestion
Add support for conditions on create and delete actions on One2Many fields.
```

File: web_action_conditionable/static/src/js/field_one2many.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                        if (!_.has(arch.attrs, item)) {
+                            return;
+                        }
+                        var expr = arch.attrs[item];
+                        try {
+                            self.activeActions[item] = py
+                                .evaluate(py.parse(py.tokenize(expr)), self.recordData)
+                                .toJSON();
+                        } catch (ignored) {
+                            // Do nothing
```
Comment (simahawk) [inline web_action_conditionable/static/src/js/field_one2many.js]: I'd log the error in the console instead.

File: web_action_conditionable/views/view.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (simahawk) [inline web_action_conditionable/views/view.xml]: can we move it to `templates/assets.xml`?

**Review by simahawk — COMMENTED** [review]
non blocking remarks

**Review by ovnicraft — APPROVED** [review]

**Review by luc-demeyer — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review :+1:

**Review by marcelsavegnago — APPROVED** [review]

**Pull request merged**